cookbook/core/containerization/multi_images.py

```diff
@@ -47,12 +47,86 @@
     It is the responsibility of the user to push a container image that matches the new name described.
 
 """
-
-# %%
-# Lets declare a task that uses an image that is derived from the defaults FQN and version
-#
-# .. todo::
-#
-#   Another example for multi-images here
-#
-# For another example where this is done please refer to :any:`intermediate_spark_dataframes_passing`
+import typing
+from typing import List
+
+import pandas as pd
+import tensorflow as tf
+from flytekit import task, workflow
+from sklearn.model_selection import train_test_split
+from sklearn.svm import SVC
+from tensorflow import keras
+
+
+@task(container_image="")
+def svm_classifier() -> float:
+    dataset_url = "https://raw.githubusercontent.com/harika-bonthu/SupportVectorClassifier/main/datasets_229906_491820_Fish.csv"
+    fish_data = pd.read_csv(dataset_url)
+
+    X = fish_data.drop(["Species"], axis="columns")
+    y = fish_data.Species
+
+    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.35)
+
+    model = SVC(kernel="linear", C=1)
+
+    model.fit(X_train, y_train)
+    svm_pred = model.predict(X_test)
+    accuracy = model.score(X_test, y_test)
+    return accuracy
+
+
+@task(container_image="")
```

Comment:
Why is the image empty

Reply:
M1 chip doesn't let me use tensorflow in requirements.in, due to which I can't generate requirements.txt, due to which I can't verify if the Dockerfile I wrote is correct. Samhita will test it on her laptop soon..

Reply:
https://github.com/tensorflow/tensorflow/issues/44751#issuecomment-1031149709 , https://stackoverflow.com/questions/70354859/anyway-to-work-with-tensorflow-in-mac-with-apple-silicon-m1-m1-pro-m1-max-gp